otherlibs/site/src/dune_site_data.mli

```diff
@@ -0,0 +1,5 @@
+(** {2 encoded} *)
+
+val hardcoded_ocamlpath : string
+
+val stdlib_dir : string
```

Comment:
In the other libraries, we version the API's of public libraries. Should we do it here as well?